configs/default/BKMN-NERO.config

```diff
@@ -1,54 +1,68 @@
-# Betaflight / STM32F7X2 (S7X2) 4.0.0
+# Betaflight / STM32F7X2 (S7X2) 4.4.3 Mar 18 2024 / 20:58:20 (738127e7e) MSP API: 1.45
 
-# this will trigger the cloud build system to include these target defines.
-#define USE_ACC
-#define USE_ACC_SPI_MPU6500
 #define USE_GYRO
-#define USE_GYRO_SPI_MPU6500
+#define USE_GYRO_SPI_ICM20602
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM20602
+#define USE_SDCARD
 
 board_name NERO
 manufacturer_id BKMN
 
-# Basic I/O
+# resources
+resource BEEPER 1 C01
+resource LED_STRIP 1 B00 # pin M5
 resource LED 1 B06
 resource LED 2 B05
 resource LED 3 B04
-resource beeper C1
+
+# Motors
+resource MOTOR 1 A00
+resource MOTOR 2 A01
+resource MOTOR 3 A02
+resource MOTOR 4 A03
+# resource MOTOR 5 B00
```

Comment:
should this be uncommented?
3.5.7: https://github.com/betaflight/betaflight/blob/e9130527cd644d549ba5f455cf053d086fa75041/src/main/target/NERO/target.c#L36

Reply:
I kept it commented because B00 can be used by led_strip,  if M5 is needed we can activate this resource and comment the led_strip

Reply:
okay, i'll let @haslinghuis decide which is best.

Reply:
Its a tradeoff either way. More likely to use the board with 4 motors with ledstrip then using 8 motors without ledstrip.